src/google.js

```diff
@@ -73,12 +74,13 @@ export async function checkSheetForChanges(sheetId) {
 /**
  * Get the data from the sheet in the initial range.
  * @param {String} sheetId - ID from the sheet from which the data should be pulled.
+ * @param {String} pageName - Name of the excel page to check
```

Comment:
It is not an Excel page. Best to have a look at what the exact name is that Google uses.